### PR TITLE
fix: medium card color on news and articles page

### PR DESCRIPTION
**Pull request by alisonjoseph**
Should be showing up light not dark.

### CODE REVIEW

**Review by jnm2377 — APPROVED** [review]

**Review by tay1orjones — APPROVED** [review]

**Review by laurenmrice — COMMENTED** [review]
- The preview link is still showing the dark cards.

- Also this may be unrelated to changing just the color of the articles in this PR, but Ariella Gilmores card if not the same height as the others. 
Screen Shot 2021-11-08 at 10 39 27 AM

**Comment by alisonjoseph** [comment]
I think there must be an upstream bug with this component, unless I'm missing something @vpicone @sstrubberg  🤔

File: src/pages/whats-happening/news-and-articles/index.mdx [hunk 1]
```diff
@@ -19,7 +19,7 @@ about design systems.
 
 ## Latest news
 
-<MediumPosts postLimit={9} />
+<MediumPosts postLimit={9} color="light" />
```
Comment (vpicone) [inline src/pages/whats-happening/news-and-articles/index.mdx:22]:
```suggestion
<MediumPosts postLimit={9} cardProps={{color: "light"}} />
```

**Review by vpicone — CHANGES_REQUESTED** [review]

**Review by laurenmrice — APPROVED** [review]
The cards are showing up as "light" now. That one card is still off size wise but it seems like a different problem to be addressed in another PR.

**Review by vpicone — APPROVED** [review]

**Pull request merged**